Library/Formula/cdrtools.rb

```diff
@@ -3,7 +3,7 @@ class Cdrtools < Formula
   homepage "http://cdrecord.org/"
 
   stable do
-    url "https://downloads.sourceforge.net/project/cdrtools/cdrtools-3.01.tar.bz2"
+    url "http://downloads.sourceforge.net/cdrtools/cdrtools-3.01.tar.bz2"
```

Comment:
I've reverted the https -> http change. Should I also evert the removal of "/project/" and "/alpha/" ?
"curl --head" shows that SourceForge responds with nice redirects which allow to just provide the name of the tarball without having to indicate the exact path (e.g. /alpha/") as long as we don't specify "/projects/". Are you OK for the simplified URLs?